src/splits.rs

```diff
@@ -119,6 +119,13 @@ pub enum Split {
     EnterWormways,
     // endregion: Wormways
 
+    // region: HuntersMarch
+    /// Enter Hunter's March (Transition)
+    ///
+    /// Splits on entering Hunter's March from the Marrow
```

Comment:
It would be good to mention in the split description that it's after the Skarrguard arena